docs/docs/configuring-ca-certificates.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Configuring CA Certificates
+---
+
+For developers using a private (or corporate) package registry that requires a certificate from a certificate authority (CA), you may need to set your certificate in your npm, yarn, or node config.
```

Comment:
I'd say "npm, yarn, and node" as if you need to configure one you'll need to configure all of them.